rancher2/resource_rancher2_node_template.go

```diff
@@ -86,30 +86,32 @@ func resourceRancher2NodeTemplateCreate(d *schema.ResourceData, meta interface{}
 }
 
 func resourceRancher2NodeTemplateRead(d *schema.ResourceData, meta interface{}) error {
-	log.Printf("[INFO] Refreshing Node Template ID %s", d.Id())
+	id := d.Id()
```

Comment:
nit: you call d.SetId() later, not sure you need this local var. maybe just use the d.Id() everywhere you have this.

Reply:
ok, updated.